test/system/comment_upload_test.rb

```diff
@@ -0,0 +1,41 @@
+require "application_system_test_case"
+
+class CommentUploadTest < ApplicationSystemTestCase
+  Capybara.default_max_wait_time = 60
+
+  test "uploading by dragging" do
+    # log in
+    visit '/'
+    click_on 'Login'
+    fill_in("username-login", with: "Bob")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
+
+    visit Node.last.path
+    drop_in_dropzone("#{Rails.root.to_s}/public/images/pl.png", false)
```

Comment:
```
ERROR["test_uploading_by_dragging", #<Minitest::Reporters::Suite:0x0000560da661a738 @name="CommentUploadTest">, 367.67084992700006]
 test_uploading_by_dragging#CommentUploadTest (367.67s)
Selenium::WebDriver::Error::JavascriptError:         Selenium::WebDriver::Error::JavascriptError: javascript error: fileList is not defined
          (Session info: headless chrome=74.0.3729.157)
          (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Linux 4.4.0-101-generic x86_64)
            test/system/comment_upload_test.rb:35:in `drop_in_dropzone'
            test/system/comment_upload_test.rb:15:in `block in <class:CommentUploadTest>'
```

Reply:
hmm, now:

```

 FAIL["test_viewing_the_dashboard", #<Minitest::Reporters::Suite:0x000055d036d2a5f8 @name="DashboardTest">, 322.3365901269999]
 test_viewing_the_dashboard#DashboardTest (322.34s)
        expected to find visible css ".row.header > h1" with text "Dashboard" but there were no matches. Also found "Community research", which matched the selector but not all filters. 
        test/system/dashboard_test.rb:19:in `block in <class:DashboardTest>'
[Screenshot]: tmp/screenshots/failures_test_uploading_by_dragging.png
ERROR["test_uploading_by_dragging", #<Minitest::Reporters::Suite:0x00007ffaf958f5b0 @name="CommentUploadTest">, 325.9385120889999]
 test_uploading_by_dragging#CommentUploadTest (325.94s)
Selenium::WebDriver::Error::JavascriptError:         Selenium::WebDriver::Error::JavascriptError: javascript error: Cannot read property 'listeners' of undefined
          (Session info: headless chrome=74.0.3729.157)
          (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Linux 4.4.0-101-generic x86_64)
            test/system/comment_upload_test.rb:32:in `drop_in_dropzone'
            test/system/comment_upload_test.rb:15:in `block in <class:CommentUploadTest>'

```

Reply:
I think this is actually because JavaScript methods aren't waiting for selectors to find things... so i'm going to insert a `assert_selector()` to get it to wait.

Reply:
If this doesn't work, we might try to do a `jQuery.ready(...` kind of thing